homeassistant/components/ecovacs/strings.json

```diff
@@ -0,0 +1,31 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_account%]"
+    },
+    "error": {
+      "invalid_auth": "Invalid username or password",
+      "unknown": "Unknown error. Please check the logs"
+    },
+    "step": {
+      "user": {
+        "data": {
+          "continent": "Continent",
+          "country": "Country",
+          "password": "[%key:common::config_flow::data::password%]",
+          "username": "E-mail or ShortID"
```

Comment:
What do you use in the ecovacs app? Can you maybe explain the email or shortId thing in the data description? 

(suggestion)

Reply:
Changed it to E-mail.
ShortId probably is only needed in China (if it is still needed) but for the most users it will be the e-mail address

Reply:
I changed it back to username as this includes all cases.